src/main/java/org/spongepowered/api/world/volume/virtual/UnrealizedBiomeVolume.java

```diff
@@ -0,0 +1,38 @@
+package org.spongepowered.api.world.volume.virtual;
+
+import org.spongepowered.api.world.biome.Biome;
+import org.spongepowered.api.world.volume.biome.BiomeVolume;
+
+public interface UnrealizedBiomeVolume<B extends BiomeVolume> extends Virtualized<Biome, B> {
```

Comment:
This isn't necessarily made use of, but it can be potentially made use by the implementation and/or plugins for "game scoped" volumes that can exist between reloads of entire registries. I did have the feeling that Schematic/ArchetypeVolume should really make use of the "virtualization" by this to further emphasize that they can be deserialized at any given point of the game scope.